### PR TITLE
Added flag for initialization scripts in yugabyted

### PR DESCRIPTION
**Pull request by baba230896**
### Summary:

- Added `--initial_scripts_dir` flag for initialization scripts functionality in `yugabyted`.
- Using the `--initial_scripts_dir` flag user can provide a directory from where yugabyted can read the scripts at the time of the first installation of the YB cluster.
- Format: 
  - For YSQL - `.sql`
  - For YCQL - `.cql`
- Scripts will be executed in the sorted name order.

### Test plan:

**1. With initialization scripts**

  - Create a directory and copy-paste the Northwind demo database files and named `1_northwind_ddl.sql` and `2_northwind_data.sql`.
  - Use [this](https://docs.yugabyte.com/latest/quick-start/explore/ycql/#1-connect-with-ycqlsh) link to create demo CQL scripts.
  - The directory will look like this:
```
ubuntu@ip-172-31-12-205:~/test-yugabyted/yugabyte-2.3.0.0$ ll ../initial-scripts/
total 352
drwxrwxr-x  2 ubuntu ubuntu   4096 Sep 18 10:58 ./
drwxrwxr-x 10 ubuntu ubuntu   4096 Sep 23 10:14 ../
-rw-rw-r--  1 ubuntu ubuntu    800 Sep 18 10:51 00_first.cql
-rw-r--r--  1 ubuntu ubuntu   6438 Sep 18 10:58 1_northwind_ddl.sql
-rw-r--r--  1 ubuntu ubuntu 339211 Sep 15 09:26 2_northwind_data.sql
```

  - Use the following to spin up the YB cluster with initialization scripts:
`bin/yugabyted start --daemon=true --ui=false --initial_scripts_dir=../initial-scripts/`

**2. Without initialization scripts**
`bin/yugabyted start --daemon=true --ui=false`

**3. With Environment variables and initialization scripts**
```
export YSQL_USER=admin
export YSQL_PASSWORD=password
export YSQL_DB=admindb
export YCQL_USER=cqladmin
export YCQL_PASSWORD=cqlpassword
export YCQL_KEYSPACE=cqlks
bin/yugabyted start --daemon=true --ui=false --initial_scripts_dir=../initial-scripts/
```

### CODE REVIEW

File: bin/yugabyted [hunk 1]
```diff
@@ -1974,6 +2008,16 @@ class YcqlProxy(object):
                 "CREATE KEYSPACE {};".format(keyspace)]
         run_process_checked(cmd)
 
+    # Runs ycqlsh with specified files.
```
Comment (iSignal) [inline bin/yugabyted:2011]: minor: add an example of the final ycql command line for files a.ycql, b.ycql

**Review by iSignal — APPROVED** [review]
Looks good. Maybe also verify that it works as expected with a docker volume mount before merging?

**Comment by baba230896** [comment]
I've checked the changes with docker volume and its working fine. Here is the example command:
```
docker run -d -v /home/ubuntu/initial-scripts:/tmp/initial-scripts yugabytedb/yugabyte:latest bin/yugabyted start --daemon=false --ui=false --initial_scripts_dir=/tmp/initial-scripts/
```

File: bin/yugabyted (hunk 1 above)
Comment (baba230896) [reply bin/yugabyted:2011]: Okay. will add

**Comment by baba230896** [comment]
@iSignal 

The initial script flag looks for the `.sql` and `.cql` format files for YSQL and YCQL respectively. 
Do we need to change with `.ysql` and `.ycql` for YSQL and YCQL respectively?

**Comment by iSignal** [comment]
.sql and .cql are fine.

**Review by iSignal — APPROVED** [review]

**Pull request merged**